scripts/package/postinstall.sh

```diff
@@ -58,7 +60,7 @@ StartLimitBurst=5
 [Service]
 Type=simple
 User=root
-Group=observiq-otel-collector
+Group=bdot
```

Comment:
Should this be `${username}` so that it is consistent? (I'm not sure if that will work, but it **should**, since it will evaluate first, then write)

Reply:
Nice catch, it will work.